src/schemas.tsx

```diff
@@ -0,0 +1,12 @@
+export interface UserData {
```

Comment:
❌ 
this is not data, it is an interface/type
so rename it to something like `UserType`